packages/arm-draw/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@datapunt/arm-draw",
+  "description": "Draw tool",
+  "license": "MPL-2.0",
+  "author": "Datapunt Amsterdam",
+  "homepage": "https://github.com/Amsterdam/amsterdam-react-maps#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Amsterdam/amsterdam-react-maps.git",
+    "directory": "packages/arm-draw"
+  },
+  "bugs": {
+    "url": "https://github.com/Amsterdam/amsterdam-react-maps/issues"
+  },
+  "version": "0.2.7",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "files": [
+    "es",
+    "lib"
+  ],
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "sideEffects": false,
+  "typings": "lib/index.d.ts",
+  "dependencies": {
+    "@datapunt/arm-core": "^0.2.7",
+    "leaflet-draw": "^1.0.4",
+    "uuid": "^8.1.0"
+  },
+  "peerDependencies": {
+    "@datapunt/asc-assets": "^0.20.0",
+    "@datapunt/asc-ui": "^0.20.0",
+    "@datapunt/react-maps": "^0.8.2-alpha.0",
+    "react": ">=16.8.6",
+    "react-dom": ">=16.8.6",
+    "styled-components": ">=4.4.1",
+    "leaflet": "^1.6.0"
```

Comment:
Should this dependency be defined here? `@datapunt/arm-core` Already depends on it, doesn't it?

Reply:
Well yes, it still needs some types and methods from leaflet